components/esp32/Kconfig

```diff
@@ -929,6 +929,25 @@ menu "ESP32-specific"
             This option depends on the CONFIG_FREERTOS_UNICORE option because RTC fast memory
             can be accessed only by PRO_CPU core.
 
+config USE_FIXED_STATIC_RAM_SIZE
```

Comment:
We are adding additional checks for Kconfig option naming soon. Please prefix these new option names with `ESP32_` so that there is no conflict with the upcoming changes.

Reply:
fixed.